src/main/kotlin/com/amazon/elasticsearch/replication/task/index/IndexReplicationTask.kt

```diff
@@ -794,6 +806,18 @@ class IndexReplicationTask(id: Long, type: String, action: String, description:
             ShardReplicationExecutor.TASK_NAME, replicationParams)
     }
 
+    override fun onIndexShardClosed(shardId: ShardId, indexShard: IndexShard?, indexSettings: Settings) {
+    }
```

Comment:
A comment would be useful as to why we are keeping this no-op.

Reply:
fixed by Sai